### PR TITLE
don't not run linux tests just because there is a doc change

### LINKED ISSUE #22120: Linux e2e tests can be inadvertently skipped
Look at https://github.com/prestodb/presto/pull/22115/files. Notice that although the linux-presto-e2e-tests seems to have passed, if you click through to the results you'll find https://app.circleci.com/pipelines/github/prestodb/presto/13223/workflows/941b1538-aaf1-4726-839a-11197d7e5ac9/jobs/51304 you'll see that it stopped with the third step, echo "Run Linux tests is false"

It updates three files, one in presto-docs and two in presto-parser.

I think the path-filtering in .circleci/config.yml does this:

          mapping: |
            presto-native-execution/.* run_native_specific_jobs true
            presto-docs/.* run_linux_tests false

since there's a presto-docs change here, run_linux_tests  is false.

However, this should only happen when the only changes are in presto-docs, not when simply one change is in presto-docs.

### PR DESCRIPTION
**Pull request by elharo**
## Description
fixes #22120

## Motivation and Context
Better to run the tests on docs changes than not run the changes on non docs changes

## Impact
no public change

## Test Plan
This PR includes a very small change in presto-docs. If linux-presto-e2e-tests runs fully on this PR (nor merely passes) then the bug is fixed. 

runlinuxtestsistrue

## Contributor checklist

- [x] Please make sure your submission complies with our [development](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#development), [formatting](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#formatting), [commit message](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#commit-formatting-and-pull-requests), and [attribution guidelines](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#attribution).
- [x] PR description addresses the issue accurately and concisely.  If the change is non-trivial, a GitHub Issue is referenced.
- [x] Documented new properties (with its default value), SQL syntax, functions, or other functionality.
- [x] If release notes are required, they follow the [release notes guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines).
- [x] Adequate tests were added if applicable.
- [ ] CI passed.

## Release Notes
Please follow [release notes guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines) and fill in the release notes below.

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -25,6 +25,5 @@ workflows:
           # <regex path-to-test> <parameter-to-set> <value-of-pipeline-parameter>
           mapping: |
             presto-native-execution/.* run_native_specific_jobs true
-            presto-docs/.* run_linux_tests false
```
Comment (elharo) [inline .circleci/config.yml:28]: Possibly we could change the default to false, and then use a regexp that matches everything except presto-docs to set run_linux_tests to true.

**Review by steveburnett — DISMISSED** [review]
LGTM! (docs) 

Pull branch, new local build, docs build fine, change in types.rst visible in local build.

File: presto-docs/src/main/sphinx/language/types.rst [hunk 2]
```diff
@@ -29,7 +29,7 @@ Integer
 ``TINYINT``
 ^^^^^^^^^^^
 
-    A 8-bit signed two's complement integer with a minimum value of
+    An 8-bit signed two's complement integer with a minimum value of
```
Comment (tdcmeehan) [inline presto-docs/src/main/sphinx/language/types.rst:32]: Can you either put this into a separate commit or raise a separate PR?

**Review by tdcmeehan — CHANGES_REQUESTED** [review]
Change looks good

File: presto-docs/src/main/sphinx/language/types.rst (hunk 2 above)
Comment (elharo) [reply presto-docs/src/main/sphinx/language/types.rst]: OK, I reverted this change for now. I don't really care about it. I just made it to make sure the CI now behaved as expected.

**Comment by elharo** [comment]
Known flake in 


[ERROR]   TestPrestoSparkNativeGeneralQueries>AbstractTestNativeGeneralQueries.testUnionAllInsert:1439->AbstractTestQueryFramework.assertQuery:159 For query: 
 SELECT * FROM tmp_presto_453cbe79d7f14f4eba01def50500754d
not equal

Rerunning

**Review by tdcmeehan — APPROVED** [review]
